File: javascript-closure-angular-client/API/Client/PaymentCalculatedTax.js
goog.provide('API.Client.PaymentCalculatedTax');

/**
 * Structure with value for each tax withhold value of this line
 * @record
 */
API.Client.PaymentCalculatedTax = function() {}

/**
 * @type {!API.Client.PaymentTaxByType}
 * @export
 */
API.Client.PaymentCalculatedTax.prototype.taxByType;

/**
 * Sum of tax type not VAT (Value Added Tax)
 * @type {!number}
 * @export
 */
API.Client.PaymentCalculatedTax.prototype.tax;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/PaymentCalculatedTax.cs
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// Structure with value for each tax withhold value of this line
    /// </summary>
    [DataContract]
    public class PaymentCalculatedTax
    {
        /// <summary>
        /// Gets or Sets TaxByType
        /// </summary>
        [DataMember(Name = "taxByType", EmitDefaultValue = false)]
        public PaymentTaxByType TaxByType { get; set; }

        /// <summary>
        /// Sum of tax type not VAT (Value Added Tax)
        /// </summary>
        /// <value>Sum of tax type not VAT (Value Added Tax)</value>
        [DataMember(Name = "tax", EmitDefaultValue = false)]
        public double? Tax { get; set; }
    }
}
